api/groups/nodeGroup.go

```diff
@@ -450,11 +450,6 @@ func (ng *nodeGroup) managedKeysWaiting(c *gin.Context) {
 // waitingEpochsLeft returns the number of epochs left for the public key until it becomes eligible
 func (ng *nodeGroup) waitingEpochsLeft(c *gin.Context) {
 	publicKey := c.Param("key")
-	if len(publicKey) == 0 {
```

Comment:
why removing this check? because it will fail anyway on the validatorPukKey decoder?

Reply:
yes.. and also calling without key "/waiting-epochs-left/` isconsidered a different path, thus the check is invalid